### PR TITLE
feat: update deps to support angular 10 (#328)

### PR DESCRIPTION
**Pull request by irustm**

### CODE REVIEW

File: projects/fullcalendar/package.json [hunk 1]
```diff
@@ -17,10 +17,11 @@
   "dependencies": {
     "@fullcalendar/core": "~5.3.0",
     "fast-deep-equal": "^3.1.1"
-  },
+  ,    "tslib": "^2.0.0"
+},
   "peerDependencies": {
-    "@angular/common": "^9.0.0",
-    "@angular/core": "^9.0.0"
+    "@angular/common": "*",
```
Comment (arshaw) [inline projects/fullcalendar/package.json:23]: @irustm , isn't it better to write `"9 - 10"` ?

**Comment by irustm** [comment]
@arshaw

**Comment by ydemartino** [comment]
Any news? FullCalendar is the only lib preventing me from upgrading to angular 10

**Comment by arshaw** [comment]
Sorry for the delay everyone. I begin merging and testing this, and found an error while building.

> An unhandled exception occurred: Cannot read property 'Minus' of undefined

https://travis-ci.com/github/fullcalendar/fullcalendar-angular/builds/179886835

The tests were passing initially on Travis, but when I cleared the node_modules dir, and when I tried it on my own machine, I get this error.

@irustm or anyone else, do you have any ideas?

**Comment by irustm** [comment]
I tried to delete everything, and installed it all over again. I didn't see any errors.

```
rm -rf node_modules
npm i
npm run test   

04 09 2020 00:11:59.222:INFO [launcher]: Launching browsers ChromeHeadless with concurrency unlimited
04 09 2020 00:11:59.227:INFO [launcher]: Starting browser ChromeHeadless
04 09 2020 00:12:15.331:INFO [Chrome Headless 85.0.4183.83 (Mac OS 10.15.4)]: Connected on socket 1pwAdrbVhf5uwAShAAAA with id 62879535
Chrome Headless 85.0.4183.83 (Mac OS 10.15.4): Executed 11 of 11 SUCCESS (1.86 secs / 1.661 secs)
TOTAL: 11 SUCCESS
TOTAL: 11 SUCCESS
```

**Comment by godly-devotion** [comment]
Just a wild suggestion, what about using a newer version of Node 12 (LTS)? Node 12.13.0 used in the previous build is about a year old now. Also I think I had problems building my project using Node 14 (latest) when I tried a while ago.

**Comment by irustm** [comment]
@arshaw fixed

**Pull request merged**